### ISSUE TITLE
Skip to end button is not keyboard focusable

### ISSUE THREAD

**Issue by r-torres-csps** [CSPS-RIW/neurolearning-vue#4]
![image](https://user-images.githubusercontent.com/76488409/214862401-066da62e-60df-4990-8d26-56a0977ac5a6.png)
I can click the 'Skip to end' button but I cannot focus on it with the keyboard.

**Comment by r-torres-csps** [comment]
I'm testing the page with NVDA and I realized that I indeed was focusing in the button, I just cannot see that I was focusing on it.

So the bug is that there is no change from normal to focus state @noamstul
